gammapy/estimators/utils.py

```diff
@@ -157,3 +155,72 @@ def estimate_exposure_reco_energy(dataset, spectral_model=None, normalize=True):
         reco_exposure = reco_exposure / ref_flux[:, np.newaxis, np.newaxis]
 
     return reco_exposure
+
+
+def _satisfies_conditions(info_dict, conditions):
+    satisfies = True
+    for key in conditions.keys():
+        satisfies &= info_dict[key.strip("_min")] > conditions[key]
+    return satisfies
+
+
+def resample_energy_edges(dataset, conditions={}):
+    """Returns energy edges that defining a binning that satisfies given conditions on the per-bin statistics.
```

Comment:
I think we always use imperative mood for docstrings. So "Return energy edges that satisfies given condition on the per bin statistics"